docs/user-guide/configure-certificates-keyring.md

```diff
@@ -0,0 +1,117 @@
+## Configuring Zowe certificates in a key ring	
+
+The JCL member `ZWEKRING` is provided as part of the PDS sample library `SZWESAMP` delivered with Zowe.  The JCL contains the security commands to create the keyring and manage the certificates that Zowe will use.  The JCL must customized before submitting and should be reviewed by a systems programmer familiar with z/OS certificates and keyrings.
+
+After running `ZWEKRING` successfully, the script `zowe-setup-certificates.sh` needs to be run which will create the `KEYSTORE_DIRECTORY` in USS.  Depending on how the `ZWEKRING` member has been customized the configuration file `zowe-setup-certificates.env` that is used to create the `KEYSTORE_DIRECTORY` will need to be customized.
+
+### ZWEKRING
+
+The `ZWEKRING` JCL helps you create a keyring containing a certificate and a local certificate authority that is used to self-sign the certificate.  Other scenarios (such as importing an existing certificate) are described in the `ZWEKRING` JCL with the commands commented out) but are currently unsupported and included in beta mode for early testing and feedback with Zowe 1.15.  If you find any issues please raise an issue at https://github.com/zowe/zowe-install-packaging/issues.  Future releases of Zowe will provide support for more keyring scenarios.  
+
+The `PRODUCT` variable specifies the z/OS security manager.  The default is `RACF` and should be changed to `ACF2` or `TSS` if you are using Access Control Facility CA-ACF2 or CA Top Secret for z/OS as your z/OS security manager.  
+
+```
+//         SET  PRODUCT=RACF         * RACF, ACF2, or TSS
+```
+
+If you are using a security manager other than RACF you will need to comment out the RACF security commands and uncomment the commands for your security manager.  This is described more in the JCL member `ZWEKRING`
```

Comment:
I don't think this is the case and no commenting is required - I believe you just change PRODUCT=ACF2 and it runs the correct DD @vit-tomica - can you confirm?

Reply:
That's correct. No need to comment out anything here.

Reply:
Good catch - thank you for the correction.  I wasn't aware that just the PRODUCT= change alone was able to run the correct DDs by itself.   Removed the offending para.